### ISSUE TITLE
Navigation parameter gets overriden at app start if AutoHighlightCorrectButton = true

### ISSUE THREAD

**Issue by MatthiasNistl** [Windows-XAML/Template10#628]
I'm using voice commands to start my app and pass different navigation parameters to the navigation service, depending on what the user said. Unfortunately, OnNavigatedToAsync of the corresponding page is fired twice, one time with the correct parameter and the second time with a parameter of null.
Steps to reproduce this in the Hamburger template: Simply add a parameter to OnStartAsync.

```
public override Task OnStartAsync(StartKind startKind, IActivatedEventArgs args)
{
    NavigationService.Navigate(typeof (Views.MainPage), "Test");
    return Task.CompletedTask;
}
```

The parameter "Test" is passed to OnNavigatedToAsync the first time, but then OnNavigatedToAsync is fired again with a parameter of null. This happens when the HamburgerMenu refreshs its buttons and only if AutoHighlightCorrectButton is set to true.

By the way, AutoHighlightCorrectButton is currently always returning true:

```
public bool AutoHighlightCorrectButton
{
    get { return true; /*(bool)GetValue(AutoHighlightCorrectButtonProperty);*/ }
    set { SetValue(AutoHighlightCorrectButtonProperty, value); }
}
```

Temporarily disabling AutoHighlightCorrectButton and re-enabling it after the first navigation should do the trick, but is there a more elegant solution?

**Comment by JerryNixon** [comment]
I look forward to your Pull Request.

**Comment by MatthiasNistl** [comment]
Fixed by Pull Request #633.
